### ISSUE TITLE
lutris beta: dxvk trouble

### ISSUE THREAD

**Issue by legluondunet** [lutris/lutris#3253]
Hello @strycore ,

could you try to reproduce this bug:

1) Wine runner system tab, declare this variable for all games:
DXVK_HUD=full
2) launch a directx9 or 11 game  with Lutris last git version and DXVK enabled:
cinematics played but:
- no DXVK HUD
- some games have just black screen with audio (directx9 games)
- some directx11 games just crash at launch (Warcraft III) 

--> I can not reproduce this bug with last lutris stable 0.5.7.1

**Comment by legluondunet** [comment]
entire log:
```

$ ./bin/lutris -d
Unable to load locale dir, translations won't work.
INFO     2020-11-03 02:11:29,754 [application.do_command_line:319]:Lutris 0.5.7.1
INFO     2020-11-03 02:11:29,755 [startup.check_driver:61]:Using NVIDIA drivers 435.21 for x86_64
INFO     2020-11-03 02:11:29,755 [startup.check_driver:65]:GPU: GeForce GTX 770
INFO     2020-11-03 02:11:29,755 [startup.check_driver:81]:GPU: 10DE:1184 1462:2825 (nvidia drivers)
DEBUG    2020-11-03 02:11:31,625 [http.get:73]:GET https://lutris.net/api/runtimes
DEBUG    2020-11-03 02:11:31,712 [http.get:73]:GET https://lutris.net/api/runtimes/icons
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
Exception ignored in: <function InputDevice.__del__ at 0x7f78471b7040>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 159, in __del__
    self.close()
  File "/usr/lib/python3/dist-packages/evdev/device.py", line 304, in close
    super().close()
  File "/usr/lib/python3/dist-packages/evdev/eventio_async.py", line 54, in close
    loop = asyncio.get_event_loop()
  File "/usr/lib/python3.8/asyncio/events.py", line 639, in get_event_loop
    raise RuntimeError('There is no current event loop in thread %r.'
RuntimeError: There is no current event loop in thread 'Thread-3'.
DEBUG    2020-11-03 02:11:46,145 [displayconfig.get_current_resolution:621]:Retrieving current resolution
DEBUG    2020-11-03 02:11:46,145 [prefix.set_virtual_desktop:202]:Enabling wine virtual desktop with default resolution of 1920x1080
DEBUG    2020-11-03 02:11:46,208 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/system32/dxgi with DXVK version
DEBUG    2020-11-03 02:11:46,209 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/system32/d3d11 with DXVK version
DEBUG    2020-11-03 02:11:46,210 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/system32/d3d10core with DXVK version
DEBUG    2020-11-03 02:11:46,210 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/system32/d3d9 with DXVK version
DEBUG    2020-11-03 02:11:46,211 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/system32/d3d12 with DXVK version
DEBUG    2020-11-03 02:11:46,212 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/dxgi with DXVK version
DEBUG    2020-11-03 02:11:46,212 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/d3d11 with DXVK version
DEBUG    2020-11-03 02:11:46,213 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/d3d10core with DXVK version
DEBUG    2020-11-03 02:11:46,214 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/d3d9 with DXVK version
DEBUG    2020-11-03 02:11:46,214 [dxvk.enable_dxvk_dll:173]:Replacing /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/d3d12 with DXVK version
DEBUG    2020-11-03 02:11:46,288 [command.start:131]:SDL_VIDEO_FULLSCREEN_DISPLAY="off"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:STEAM_RUNTIME="/home/romain/.local/share/lutris/runtime/steam"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:LD_LIBRARY_PATH="/home/romain/.local/share/lutris/runners/wine/wine-staging-5.7-x86_64/lib:/lib64:/usr/lib/x86_64-linux-gnu/libfakeroot:/usr/lib/i386-linux-gnu/sse2:/usr/lib/i386-linux-gnu/i686/sse2:/lib:/usr/local/lib:/usr/lib32:/lib32:/lib/i386-linux-gnu:/lib/x86_64-linux-gnu:/usr/lib:/usr/lib/i386-linux-gnu:/usr/lib/x86_64-linux-gnu:/home/romain/.local/share/lutris/runtime/lib32:/home/romain/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:/home/romain/.local/share/lutris/runtime/steam/i386/lib:/home/romain/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:/home/romain/.local/share/lutris/runtime/steam/i386/usr/lib:/home/romain/.local/share/lutris/runtime/lib64:/home/romain/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:/home/romain/.local/share/lutris/runtime/steam/amd64/lib:/home/romain/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:/home/romain/.local/share/lutris/runtime/steam/amd64/usr/lib:$LD_LIBRARY_PATH"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:DXVK_HUD="full"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:WINEDEBUG="-all"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:WINEARCH="win64"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:WINE="/home/romain/.local/share/lutris/runners/wine/wine-staging-5.7-x86_64/bin/wine"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:WINEPREFIX="/home/romain/Games/tomb-raider-ii"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:WINEESYNC="0"
DEBUG    2020-11-03 02:11:46,289 [command.start:131]:WINEFSYNC="0"
DEBUG    2020-11-03 02:11:46,290 [command.start:131]:WINE_LARGE_ADDRESS_AWARE="1"
DEBUG    2020-11-03 02:11:46,290 [command.start:131]:game_name="Tomb Raider II - GOG"
DEBUG    2020-11-03 02:11:46,290 [command.start:131]:PYTHONPATH="/home/romain/Autres_applications/Jeux/Lutris/lutris:/home/romain/Autres_applications/Jeux/Lutris/lutris/bin:/usr/lib/python38.zip:/usr/lib/python3.8:/usr/lib/python3.8/lib-dynload:/usr/local/lib/python3.8/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.8/dist-packages"
DEBUG    2020-11-03 02:11:46,290 [command.start:132]:/home/romain/Autres_applications/Jeux/Lutris/lutris/share/lutris/bin/lutris-wrapper Tomb Raider II - GOG 0 0 gamemoderun /home/romain/.local/share/lutris/runners/wine/wine-staging-5.7-x86_64/bin/wine /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/wineconsole.exe c:\TR2M.BAT
Running gamemoderun /home/romain/.local/share/lutris/runners/wine/wine-staging-5.7-x86_64/bin/wine /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/wineconsole.exe c:\TR2M.BAT
Initial process has started with pid 10929
Start monitoring process.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
user.reg:733: Malformed value '"HDA NVidia HDMI/DP,pc"=3 (js)"="disabled"'
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
Initial process has exited (return code: 0)
Monitored process exited.
All monitored processes have exited.
Exit with returncode 0
DEBUG    2020-11-03 02:12:06,559 [command.on_stop:180]:Process hasn't terminated yet
DEBUG    2020-11-03 02:12:06,559 [command.on_stop:182]:Process 10927 has terminated with code 0
DEBUG    2020-11-03 02:12:09,906 [game.beat:536]:Game thread stopped
WARNING  2020-11-03 02:12:09,906 [game.on_game_quit:566]:Game still running (state: running)
INFO     2020-11-03 02:12:09,906 [game.stop:551]:Stopping Tomb Raider II - GOG (wine)
DEBUG    2020-11-03 02:12:09,942 [game.on_game_quit:587]:Tomb Raider II - GOG stopped at mar., 03 nov. 2020 02:12:09
DEBUG    2020-11-03 02:12:09,942 [game.save:261]:Saving Tomb Raider II - GOG (wine) with config ID tomb-raider-ii-gog-dgvoodoo2-1586994145
```

**Comment by legluondunet** [comment]
seems related to Commit dafadf9f5b3002262af47e9e005067a287c221b7, if you revert it, issue is gone.

**Comment by strycore** [comment]
Fixed by #3256
